ui/Screen/Org/Projects.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="typescript">
+  import type { Project } from "ui/src/project";
+
+  import AnchorList from "ui/Screen/Org/AnchorList.svelte";
+
+  import type * as project from "ui/src/project";
+  import type * as theGraphApi from "ui/src/theGraphApi";
```

Comment:
I’d prefer if we’d only need to import one module for everything org related. This doesn’t mean the all code has to live in `ui/src/org`. It could be split into multiple submodules.

Reply:
d0d775bd8f7bd51643d51993d0da7165413d9344